Core/Security/X509Certificate.py

```diff
@@ -2,20 +2,52 @@
 """
 __RCSID__ = "$Id$"
 
-import GSI
+import M2Crypto
+import asn1
+import datetime
+
 import os
 from DIRAC import S_OK, S_ERROR
 from DIRAC.Core.Utilities import Time
 from DIRAC.Core.Utilities import DErrno
 from DIRAC.ConfigurationSystem.Client.Helpers import Registry
 
+VOMS_EXTENSION_OID = '1.3.6.1.4.1.8005.100.100.5'
+VOMS_FQANS_OID = '1.3.6.1.4.1.8005.100.100.4'
+VOMS_GENERIC_ATTRS_OID = '1.3.6.1.4.1.8005.100.100.11'
+DOMAIN_COMPONENT_OID = '0.9.2342.19200300.100.1.25'
+ORGANIZATIONAL_UNIT_NAME_OID = '2.5.4.11'
+COMMON_NAME_OID = '2.5.4.3'
```

Comment:
Oh... shit! What's all above? Where does it come from? There's definitely the need for some documentation here!